src/displayCommentPopup.js

```diff
@@ -30,25 +34,52 @@ const displayCommentpopup = (id, name, type, description, race, image) => {
       </div>
       <hr>
       <div class="popup-comment-section d-flex flex-column align-items-start">
-        <h4 class="align-self-center text-white font-30px">Comments</h4>
+        <h4 class="comments-container-title align-self-center text-white font-30px">Comments</h4>
         <div class="popup-comments-container p-3 d-flex flex-column align-self-stretch"></div>
       </div>
       <hr>
-      <div class="popup-comments-form-container d-flex flex-column">
+      <div class="popup-comments-form-container d-flex flex-column p-3">
         <h4 class="align-self-center text-white font-30px">Add a comment!</h4>
-        <div class="popup-comments-form"></div>
+        <div class="popup-comments-form d-flex flex-column p-3">
+          <input type="text" class="comment-user mb-3 w-50" name="contact-user" placeholder="Username" maxlength="30" required>
+          <textarea class="comment-message mb-3" name="contact-message" placeholder="Your insights" maxlength="500" required></textarea>
+          <button type="button" class="add-comment-btn align-self-start">Post Comment</button> 
+        </div>
```

Comment:
I tested `autofocus` on this but it didn't seem to work.
Maybe it only works for elements inside a form.
Right now the popup comments look great and work fine.